xgcm/grid.py

```diff
@@ -1361,24 +1361,42 @@ def iterate_axis_combinations(items):
         # return the product of the metrics
         return functools.reduce(operator.mul, metric_vars, 1)
 
-    def interp_like(self, da_source, metric):
+    @docstrings.dedent
+    def interp_like(self, da_target, da_source):
+        """Compares positions between two data arrays and interpolates missing values
```

Comment:
I find this description a bit misleading. To me it sounds like this interpolates gaps in the the data. How about: "Compares positions between two dataarrays and interpolates `da_source` to the position of `da_target` if necessary."